packages/react-compose/etc/react-compose.api.md

```diff
@@ -115,6 +123,14 @@ export interface ShorthandConfig<TProps> {
     mappedProp?: keyof TProps;
 }
 
+// Warning: (ae-forgotten-export) The symbol "ReactNode" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type SlotProp<TProps> = ReactNode | ObjectSlotProp<TProps>;

```

Comment:
I'm calling this `SlotProp`. It's a prop for a slot & props for the slot. Little unsure about the name but I like it more than the term `shorthand`. Via compose, we are exposing term such as slot/slotProps. We should try keeping the names close